tests/sles4sap/hana_install.pm

```diff
@@ -41,18 +43,23 @@ sub run {
     # Add host's IP to /etc/hosts
     $self->add_hostname_to_hosts;
 
+    # Install libopenssl1_0_0 for older (<SPS03) HANA versions on SLE15+
+    install_libopenssl_legacy($path);
```

Comment:
It will need `$self->` as the first argument to the method in `lib/sles4sap` is that structure itself. So either add `$self->` to the call here, or better yet, remove `$self` from L422 of `lib/sles4sap`.

Reply:
Yes, and I also forget to add `add_suseconnect_product` in `sles4sap.pm`...

Better to add `$self->` here I think, to be coherent with the other functions calls.